lib/api/auth/authHelpers.ts

```diff
@@ -3,21 +3,22 @@ import { db } from "@/lib/api/db";
 import { getErrorMessage } from "../apiHelpers";
 
 export const USER_ID = "ROOT_USER_afuqioweruwnvasf";
+export const ADMIN_USERNAME = "admin";
 
 const SALT = 15;
 
 export const hashPassword = async (password: string): Promise<string> => {
     return await hash(password, SALT);
 };
 
-export const checkIfPasswordHasBeenSet = async (): Promise<boolean> => {
+export const checkIfAdminExists = async (): Promise<boolean> => {
     try {
-        const existingPwd = await db.user.findUnique({
+        const existingAdmin = await db.user.findUnique({
             where: {
-                username: USER_ID,
+                username: ADMIN_USERNAME,
```

Comment:
only issue here is that is_super_admin is not explicitly labeled as a unique field in our schema currently so it's a bit tricky to put that in the where clause. I agree that it would be better than checking username - I've changed it to say find_first where is_super_admin is true but that doesn't feel 100% perfect either

Reply:
Yeah that's fair, not sure what to do about it